### ISSUE TITLE
Provide alternative text for images

### ISSUE THREAD

**Issue by eelzi-ONRR** [DOI-ONRR/nrrd#3468]
OCIO 2024 Report

Violation: Provide alternative text for images

Instances: 3
Occurrence: <1%
Severity: 10
Priority:

Notes/causes: This svg element does not have a mechanism that allows an accessible name value to be calculated

Location: https://revenuedata.doi.gov/fact-sheet

**Comment by lpgoldstein** [comment]
onrr logo should be marked decorative and not have alt text
Disbursements bar chart doesn't have alt text

**Comment by eelzi-ONRR** [comment]
@lpgoldstein Logos are not considered decorative and they do need alt text

https://www.w3.org/WAI/standards-guidelines/act/rules/qt1vmo/

https://sc.edu/about/offices_and_divisions/digital-accessibility/toolbox/best_practices/alternative_text/logo-alt-text/index.php#:~:text=Logos%20and%20marks%20may%20seem,text%20for%20screen%20reader%20users.

**Comment by lpgoldstein** [comment]
@eelzi-ONRR Thanks I was checking with ANDI and that was one of the issues that popped up.

**Comment by eelzi-ONRR** [comment]
According to the OCIO report it's the stacked bar charts causing the issue - you can find them by using inspect and searching for 0 0 436 130

**Comment by eelzi-ONRR** [comment]
The problem ANDI is marking isn't that it's decorative - it's that this image is an svg and svgs should have a title or ID instead of an alt text tag

https://accessibilityinsights.io/info-examples/web/svg-img-alt/

**Comment by lpgoldstein** [comment]
@eelzi-ONRR Thanks for clarification. I'll keep investigating and then propose solutions.

**Comment by lpgoldstein** [comment]
@greensomes We need some help with this one next sprint - we need to hide the svgs of each individual bar in each chart - there are 72 instances. I'd like to learn how to do this if possible. Also the alt text for the first chart - disbursements isn't showing up.

**Comment by lpgoldstein** [comment]
@greensomes looks great, thanks!
